src/components/DateRangePicker.jsx

```diff
@@ -280,7 +280,7 @@ class DateRangePicker extends React.Component {
 
     if (withPortal || withFullScreenPortal) {
       return (
-        <Portal isOpened>
+        <Portal>
```

Comment:
can you elaborate on what the effective change will be without `isOpened`?

Reply:
in v4, the portal is rendered on cDM, so I don't think the isOpened is necessary. If react-dates needs this info and more control over it (which it doesn't seem like it does), then we could use PortalWithState. @tajo should definitely confirm this though.